konduit-serving-pipeline/src/main/java/ai/konduit/serving/pipeline/api/process/ProcessUtils.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * *****************************************************************************
+ * Copyright (c) 2020 Konduit K.K.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * ****************************************************************************
+ */
+
+package ai.konduit.serving.pipeline.api.process;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.io.IOUtils;
+import org.nd4j.common.base.Preconditions;
+import org.nd4j.shade.guava.collect.Streams;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Slf4j
+public class ProcessUtils {
+
+    private static String OS = System.getProperty("os.name").toLowerCase();
+
+    public static boolean isWindows() {
+        return OS.contains("win");
+    }
```

Comment:
I would recommend zt-exec here instead of doing things like this.

Reply:
Ah yes! Thanks for suggesting this. It really makes it easy to work with processes. I was trying to keep the `konduit-serving-pipeline` module clean from extra dependencies since I've already broken a rule by adding the ProcessUtils logic in there.